### PR TITLE
Update inner_pro.py

### PR DESCRIPTION
**Pull request by bhs5567**
I've added a function for finding factorial.

### CODE REVIEW

File: inner_pro.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-responses = ["welcome to smart calculator", "my name is Samani Bhavesh but you can call me as SB", "Thanks",
+responses = ["welcome to a smart calculator", "my name is Bhavesh but you can call me as SB.", "Thanks",
```
Comment (Bhaveshs111) [inline inner_pro.py:1]: ok

Comment (bhs5567) [reply inner_pro.py:1]: Wc

File: inner_pro.py [hunk 2]
```diff
@@ -41,6 +41,15 @@ def hcf(lst):  # GCD
                 if s == len(lst):
                     return (small)
         small -= 1
+        
+def facto(lst):
+  n=lst[0]
```
Comment (Bhaveshs111) [inline inner_pro.py:46]: fact(n) ?

File: inner_pro.py [hunk 3]
```diff
@@ -44,11 +44,17 @@ def hcf(lst):  # GCD
         
 def facto(lst):
   n=lst[0]
+  ans=fact(n)
+  return ans
+
```
Comment (Bhaveshs111) [inline inner_pro.py:49]: thank you for your suggestion,
but it's still not working.

**Pull request merged**